### DISCUSSION TITLE
Does Birdnet work with Boot to CLI

### DISCUSSION THREAD

**Discussion by PeterIP4** [mcguirepr89/BirdNET-Pi#496] [Q&A] [answered]
Currently I boot to the graphic interface. I understand this uses more power than the text/CLI interface.

If I boot to the text/CLI interface will birdnet still work.

**Comment by mcguirepr89** [comment, marked as answer]
Indeed it does!! In fact, if you use the [installation guide](https://github.com/mcguirepr89/BirdNET-Pi/wiki/Installation-Guide), you don't install the desktop at all -- I recommend it, especially if you're going to be booting into `multi-user.target` (CLI).

My best,
Patrick

**Reply by PeterIP4** [reply]
Thanks. I'm not familiar with the cli commands so I tend to install the graphical interface which is more intuitive. I guess I'll just have to learn as I go. Hopefully this will reduce the power I use.
